docs/tutorials/onnx/export_mxnet_to_onnx.md

```diff
@@ -0,0 +1,100 @@
+
+# Exporting MXNet model to ONNX format
+
+[Open Neural Network Exchange (ONNX)](https://github.com/onnx/onnx) provides an open source format for AI models. It defines an extensible computation graph model, as well as definitions of built-in operators and standard data types.
+
+In this tutorial, we will show how you can save MXNet models to ONNX format.
+
+Current MXNet-ONNX import and export operator support and coverage can be found [here](https://cwiki.apache.org/confluence/display/MXNET/ONNX+Operator+Coverage).
+
+In this tutorial we will:
+    
+- learn how to use MXNet to ONNX exporter on pretrained models
+
+## Pre-requisite
+
+To run the tutorial you will need to have installed the following python modules:
+- [MXNet == 1.3.0](http://mxnet.incubator.apache.org/install/index.html)
+- [onnx](https://github.com/onnx/onnx) v1.2.1 (follow the install guide)
+MXNet ONNX importer and exporter follows version 7 of ONNX operator set.
+
+
+```python
+import mxnet as mx
+import numpy as np
+from mxnet.contrib import onnx as onnx_mxnet
+import logging
+logging.basicConfig(level=logging.INFO)
+```
+
+## Downloading a model from the MXNet model zoo
+
+We download a pre-trained model, in this case resnet-18 model, trained on [ImageNet](http://www.image-net.org/) from the [MXNet Model Zoo](http://data.mxnet.io/models/imagenet/).
+We will also download synset file to match labels.
+
+```python
+# Download pretrained resnet model - json and params from mxnet model zoo.
+path='http://data.mxnet.io/models/imagenet/'
```

Comment:
So does this do the downloading for us later? The earlier statement made it sound like I need to go to the model zoo and manually download the model.

Reply:
we will download the files here. fixed comment